doc/adrs/003_add_support_for_unsynchronised_diagrams.adoc

```diff
@@ -0,0 +1,80 @@
+= ADR-003 - Add support for unsynchronized diagrams
+
+== Context
+
+Sirius RCP supports unsynschronized diagrams in which a diagram can have some of its mappings defined as unsynchronized.
+An unsynchronized mapping will not automatically trigger the appearance of a figure in the diagram if a semantic element compatible with the mapping is found.
+The figure will only appear if some tool has explicitely requested the creation of a view for this mapping and semantic element thanks to the `Create View` model operation.
+In a similar fashion, it is possible to delete a figure from the diagram without having to delete its semantic element by using the `Delete View` model operation.
+
+Sirius RCP also supports unsynchronizable mappings which, contrary to regular unsynchronized mappings, give the end users some control over the synchronization policy.
+
+We want to support unsynchronized descriptions in Sirius Components.
+In order to refresh unsynchronised descriptions in a diagram, we need to know which diagram elements have been previously created for unsynchronized descriptions in order to keep them on screen.
+This ADR is thus built on top of the previous ADR which introduces the ability to create a diagram using an existing diagram to perform an incremental refresh.
+
+
+== Decision
+
+Node and edge descriptions will now contain a synchronization policy.
+This policy will support synchronized descriptions, the default behavior, and unsynchronized descriptions.
```

Comment:
Maybe I missed something, but shouldn't the compatibility layer's `*Converter` be updated to get this information from the VSMs into the new API?

Reply:
Yes that will be necessary to in the future, this PR is really about the introduction of the raw API. In my mind, as said in the ADR, such odesign support will have to be done with the support of Create View and Delete View otherwise it's not useful. Just like Hugo, I did not try to cover that in this PR on purpose.